starfyre/compiler.py

```diff
@@ -56,10 +74,28 @@ def remove_empty_lines_from_end(lines):
 
     # regex pattern to match if a line is a css import, e.g. import "style.css"
     css_import_pattern = re.compile(r"^import\s[\"\'](.*?\.css)[\"\']")
-    
+
     with open(fyre_file_name, "r") as fyre_file:
         for line in fyre_file.readlines():
             css_import_match = css_import_pattern.search(line)
+
+            # check for fyre import styles
+            has_fyre_import = check_import_line(line=line, project_dir=project_dir)
+
+            # If the line is a fyre import statement, modify it to ensure proper resolution
+            if has_fyre_import:
+                # Split the line into module part and imported component
+                module_part, imported_component = line.split(" import ")
+
+                # Extract the file name to import from the module part
+                file_to_import = module_part.split(".")[-1]
+
+                # Get the name of the project directory
+                project_dir_name = str(project_dir).split("/")[-1]
+
+                # Modify the line to use the resolved import path
+                line = f'from {project_dir_name}.build.{file_to_import} import {imported_component}'
```

Comment:
Let's not do this. We don't want to change any import paths. Every import should work as they are mentioned. We need or atleast work with `build` as the the top directory. Not using `project_dir_name`

Reply:
Thanks @sansyrox ,
I don't really understand this part:
> We need or atleast work with build as the the top directory. Not using project_dir_name
 Please can you explain further?

Reply:
@DeleMike , all I am saying is that we should not be changing this to `{project_dir_name}.build` can we not keep the regular import structure and then make it work?

or if that doesn't work, we should use `build.{file_to_import}`

Reply:
I'm trying to understand, Do you mean that we don't want the `project_dir_name` to exist in the result of the transpiled file?

Do you have any tips on what the transpiled import should be like and how it should work too?

Reply:
Alright, I will look into this.